### PR TITLE
Fix gif and css

### PR DESCRIPTION
**Pull request by seanjyjy**

### CODE REVIEW

File: src/contactus/body/styles.css [hunk 1]
```diff
@@ -125,7 +125,7 @@ textarea:focus {
 
 .form-details-error-holder > span {
   align-self: center;
-  color: #cc0000;
+  color: #cc00;
```
Comment (keanecjy) [inline src/contactus/body/styles.css]: Shouldn't this be #c00?

File: src/contactus/body/styles.css [hunk 2]
```diff
@@ -90,6 +66,30 @@ textarea:focus {
...
+}
+
+.form-particular-error-holder {
+  height: 60px;
+  width: 15%;
+}
+
+.form-particular-error-holder > span {
+  align-self: center;
+  color: #cc00;
```
Comment (keanecjy) [inline src/contactus/body/styles.css]: Shouldn't this be #c00?

**Review by keanecjy — DISMISSED** [review]
Some nits to fix.

**Review by Ashley-Lau — APPROVED** [review]
LGTM

**Pull request merged**